packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -285,6 +287,12 @@ export class DevWorkspaceClient extends WorkspaceClient {
     devworkspace.spec.started = false;
     const createdWorkspace = await DwApi.createWorkspace(devworkspace);
 
+    this.addEnvVarsToContainers(
+      createdWorkspace.spec.template.components,
+      pluginRegistryUrl,
+      pluginRegistryInternalUrl,
+    );
+
```

Comment:
```
devworkspace.spec.template.components = [];
```

https://github.com/eclipse-che/che-dashboard/pull/522/files#diff-e8e0e0be554873a4658a2f3c8863a28427a93bb78e94a7ec08550311bfc7d456R281

I think it will be an empty array. 

WDYT?

Reply:
good catch, 
adding a fixup